WORKSPACE

```diff
@@ -14,6 +14,7 @@
 #
 ################################################################################
 #
+workspace(name = "io_istio_proxy")
```

Comment:
Educate me: what is the purpose?

Reply:
Every workspace must be named in bazel. Not sure why this was missing.

Reply:
@lambdai it's to force importers of this workspace to use the same common name, to avoid duplicates.

Reply:
get it!